tensorflow_datasets/core/visualization.py

```diff
@@ -28,6 +27,18 @@
 from tensorflow_datasets.core import lazy_imports_lib
 
 
+matplotlib = lazy_imports_lib.LazyImporter.matplotlib
+
+def draw_text(ax, xy, txt, sz=14):
+    text = ax.text(*xy, txt, verticalalignment='top', color='white', fontsize=sz, weight="normal",bbox=dict(facecolor='black', alpha=0.7, pad=0.3))
```

Comment:
Could you format your code using https://www.tensorflow.org/datasets/add_dataset#5_check_your_code_style